### PR TITLE
Update troubleshooting-tentacles.md

### PR DESCRIPTION
**Pull request by paraicoceallaigh**

### CODE REVIEW

File: src/pages/docs/infrastructure/deployment-targets/tentacle/troubleshooting-tentacles.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ navOrder: 60
 
 All of the 'classic' problems of TCP networking: firewalls, proxies, timeouts, DNS issues, and so-on can affect Octopus Tentacles. This guide will help to track down these issues when either a machine cannot be "Discovered" (Listening Tentacles) or "Registered" (Polling Tentacles) with the Octopus Server, or a previously working machine fails a health-check with errors from the networking stack.
 
+:::div{.problem}
+**WARNING** 
+A breaking change in Tentacle releases with version 6.3+ means that all versions above 6.2.277 will require .NET 4.8 or above to run. This is a Microsoft dependency due to EOL for older .NET versions.
```
Comment (jbristowe) [inline src/pages/docs/infrastructure/deployment-targets/tentacle/troubleshooting-tentacles.md:14]:
```suggestion
A breaking change was made in Octopus Tentacle 6.2.277 that requires version 4.8 (or greater) of .NET runtime. This change was made as the result of an end-of-line (EOL) policy by Microsoft for older versions of the .NET runtime.
```

Comment (paraicoceallaigh) [reply src/pages/docs/infrastructure/deployment-targets/tentacle/troubleshooting-tentacles.md:14]: Yeah John I believe its after 6.2.277t hat .NET 4.8 is required - https://octopus.com/blog/tentacle-dotnet-version-change#what-if-i-cant-upgrade-my-deployment-targets

Comment (steve-fenton-octopus) [reply src/pages/docs/infrastructure/deployment-targets/tentacle/troubleshooting-tentacles.md:14]: @paraicoceallaigh I've adjusted the edit for your review.

Please can you also complete the CLA check. Once you've done it, it won't prompt for that again.

Thanks

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM!

**Pull request merged**